appveyor.yml

```diff
@@ -4,14 +4,15 @@ clone_folder: c:\gopath\src\github.com\kasvith\kache
 
 environment:
   GOPATH: c:\gopath
+  GO111MODULE: on
 
-stack: go 1.10
+stack: go 1.11
 
 before_test:
   - set PATH=%GOPATH%\bin;c:\go\bin;%PATH%
   - go version
   - go env
-  - go get github.com/magefile/mage
+  - env "GO111MODULE=off" go get github.com/magefile/mage
```

Comment:
can you move `go get github.com/magefile/mage` to a new line like

`- go get github.com/magefile/mage`

Reply:
I want to set the env just for this specific command so it can't be moved to another line, by turning module support `off` we install mage on `$GOPATH/bin` which will be available for us on our `$PATH` to use on later commands